test/refactoring/test_refactor.py

```diff
@@ -175,3 +175,69 @@ def test_refactor_bad_modify_count(default_shared_state, hst_data, caplog, capsy
         assert line in out1
     assert rmap_chk2 is False
     assert out2 in expected2
+
+
+@mark.refactoring
+@mark.refactor
+@mark.or_bars
+def test_or_bars_refactor_add_file(jwst_serverless_state, jwst_data, test_temp_dir, caplog, capsys):
+    files = f"{jwst_data}/jwst_miri_ipc_0002.rmap \
+        {test_temp_dir}/jwst_miri_ipc_0003.add.rmap \
+            {jwst_data}/jwst_miri_ipc_0003.add.fits"
+    with caplog.at_level(logging.INFO, logger="CRDS"):
+        RefactorScript(f"crds.refactor insert {files}")()
+        out = caplog.text
+    expected = """Inserting jwst_miri_ipc_0003.add.fits into 'jwst_miri_ipc_0002.rmap'
+Setting 'META.INSTRUMENT.BAND [BAND]' = 'UNDEFINED' to value of 'P_BAND' = 'SHORT | MEDIUM |'
+Setting 'META.INSTRUMENT.DETECTOR [DETECTOR]' = 'MIRIMAGE' to value of 'P_DETECT' = 'MIRIFUSHORT|MIRIFULONG|'
+0 errors
+3 infos""".splitlines()
```

Comment:
Consider removing the infos reporting. causes issue when running not in a container.